src/core/ActivityInfoTree.cpp

```diff
@@ -283,5 +302,70 @@ void ActivityInfoTree::init()
 
     qmlRegisterSingletonType<QObject>("GCompris", 1, 0, "ActivityInfoTree", menuTreeProvider);
     qmlRegisterType<ActivityInfo>("GCompris", 1, 0, "ActivityInfo");
+
+
+}
+void ActivityInfoTree::filterBySearch(const QString& text,int cursorPosition)
```

Comment:
any other way through which we can access cursor position in activityinfotree.cpp? Using q_property ?